docs/gitbook/reference/interbit/README.md

```diff
@@ -2,9 +2,9 @@
 
 Interbit offers the same features as the `interbit-cli` but in small functions requireable for programmatically managing your chains.
 
-The interbit package contains function to help you 
+The Interbit package contains function to help you
```

Comment:
The `interbit` package. Package names are case sensitive! This applies across this file. Capitalize if you mean Ineterbit the product, but not if you mean the package.